test/stream-change-remote.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * for this test to pass, you must relay 127.0.0.2:8082 to 127.0.0.1:8082
+ *
+ * this python script works:
+ *
+ * r = socket(AF_INET, SOCK_DGRAM)
+ * r.bind(("127.0.0.2", 8082))
+ *
+ * s = socket(AF_INET, SOCK_DGRAM)
+ * s.connect(("127.0.0.1", 8082))
+ *
+ * while True:
+ *     buf = r.recv(2000)
+ *     s.send(buf)
+ */
```

Comment:
You can use the `udx_stream_relay_to()` function for setting up similar relaying with just UDX streams.

Reply:
suggestion taken, test updated